conu/utils/core.py

```diff
@@ -100,14 +100,17 @@ def raw(self):
 class Probe(object):
     def check_port(self, port, host="127.0.0.1", timeout=2):
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        sock.settimeout(timeout)
-        result = sock.connect_ex((host, port))
-        if result == 0:
-            logger.debug('port OPENed: %s:%s' % (host, port))
-            return True
-        else:
-            logger.debug('port CLOSEDed: %s:%s' % (host, port))
-            return False
+        try:
+            sock.settimeout(timeout)
+            result = sock.connect_ex((host, port))
+            if result == 0:
+                logger.debug('port OPENed: %s:%s' % (host, port))
```

Comment:
please improve the log messages, at least something like `port is opened: %s...`, `port is closed: %s...`

Reply:
(`port CLOSEDed` shreds my eyes)

Reply:
Sure, I didn't actually touch the log messages in this PR, but I can fix them.